doc/releases/changelog-0.28.0.md

```diff
@@ -547,11 +370,14 @@
          [-0.38466667, -0.19233333,  0.        ,  0.        ,  0.19233333]])>
   ```
 
-* Thy PyTorch interface supports the new return system and users can use jacobian and hessian using custom differentiation
-  methods (e.g., parameter-shift, finite difference or adjoint).
+* The PyTorch interface is now fully supported when `qml.enable_return()` has been called, allowing the calculation of Jacobian and Hessian using custom differentiation methods (e.g., parameter-shift, finite difference, or adjoint).
   [(#3416)](https://github.com/PennyLaneAI/pennylane/pull/3414)
   
   ```python
+  import torch
+
+  qml.enable_return()
+
   dev = qml.device("default.qubit", wires=2)
 
   @qml.qnode(dev, diff_method="parameter-shift", interface="torch")
```

Comment:
There is a typo on line 387 with an extra `>`

Reply:
Just making sure you're paying attention. Totally left that there on purpose